webapp/web/configuration.css

```
@@ -132,19 +103,19 @@
     right: 15px;
 }
 
-.configure-package__tag-responses-item-drag-handle-0 {
+.tag-responses__item-drag-handle-0 {
     top: 20px;
 }
 
-.configure-package__tag-responses-item-drag-handle:hover {
+.tag-responses__item-drag-handle:hover {
     cursor: grab;
 }
 
-.configure-package__tag-responses-item-drag-handle-dragging {
+.tag-responses__item-drag-handle_dragging {
```

should this be:
```suggestion
.tag-responses__item-drag-handle--dragging {
```
?

---

Yes, it's a modifier. Corrected.